printer_mods/DoubleT/switchwire_tft_display/readme.md

```diff
@@ -0,0 +1,35 @@
+# 3.5in TFT Display Front Skirt - Switchwire
+
+This mode replaces the front middle skirt in order to mount a 3.5in touchscreen display for Klipperscreen. It has two printed parts: the main body and the front plate. Threaded inserts go into the main body and the M3x8 screws are used to mount the front plate onto the main body.
+
+![image1](./images/image1.png)
+
+![image2](./images/image2.png)
+
+This is a remix of the original display mount by `rcfishhunt`.
+
+## BOM
+
+- 5 threaded inserts (stock)
+- 4 M3x8 BHCS
+- [3.5in TFT Display](https://www.amazon.com/gp/product/B07KLDKM5D/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1)
```

Comment:
Please remove extraneous stuff from the URL, should be just

    https://www.amazon.com/gp/product/B07KLDKM5D